metrics/network/remote_network/README.md

```diff
@@ -0,0 +1,39 @@
+# Clear Containers remote networking test
+
+- `remote networking` measures bandwidth using iperf3 using a remote setup. Host A will run 
```

Comment:
@GabyCT I'm not sure which makes more sense, but I should have been more clear in my initial comment. Here is the change I was suggesting:
The `remote-networking-iperf3.sh` script measures bandwidth using `iperf3` using...